rest-server/src/config/express.js

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import helmet from 'helmet';
+import router from '../router';
+const middleWare = [
```

Comment:
If this grows, should we just make a middleware folder and file for all of this?